python/sdist/custom_commands.py

```diff
@@ -139,6 +155,25 @@ def run(self):
 class my_build_ext(build_ext):
     """Custom build_ext to allow keeping otherwise temporary static libs"""
 
+    def build_extension(self, ext):
+        # do replacements here
+        compilerType = self.compiler.compiler_type
+        # this requires more work
+        #try: ext.extra_compile_args = ext.extra_compile_args, ext.(extra_compile_args_+compilerType)]
```

Comment:
Need to look in more detail what the problem is there. Should be possible to get it running...

Reply:
Can you shortly explain was (not) working there?

Reply:
Soll ich es in Kürze erklären?
When the constructor for the `Extension` class is called in `setup.py`, and the attribute `extra_compile_args_mingw32` is included, there is no error message, but when the class is referenced in `my_build_ext`, the attribute is missing. It appears that the `Extension` class ignores all attributes that are not part of `Extension.__init__`.

Reply:
done